#numpy.ndarray data	
int16[] data 	# [pixX, pixY, theta]